crates/sui-framework/packages/bridge/sources/committee.move

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+#[allow(unused_use)]
+module bridge::committee {
+    use std::vector;
+
+    use sui::ecdsa_k1;
+    use sui::vec_map::{Self, VecMap};
+
+    friend bridge::bridge;
+
+    const ESignatureBelowThreshold: u64 = 0;
+
+    struct BridgeCommittee has store {
+        // commitee pub key and weight
+        pub_keys: VecMap<vector<u8>, u64>,
+        threshold: u64
+    }
+
+    public(friend) fun create_genesis_static_committee(): BridgeCommittee{
+        BridgeCommittee{
+            pub_keys: vec_map::empty<vector<u8>, u64>(),
+            threshold: 0
+        }
+    }
+
+    public fun verify_signatures(
+        committee: &BridgeCommittee,
+        message: vector<u8>,
+        signatures: vector<vector<u8>>
+    ) {
+        let (i, signature_counts) = (0, vector::length(&signatures));
+        let threshold = 0;
+        while (i < signature_counts) {
```

Comment:
we should check a pubkey should not appear twice in `signatures` and add a test for it